### PR TITLE
EE - EP Merge Prototype: Add hoppy config and hoppy request / response models

### PR DESCRIPTION
**Pull request by dfitchett**
### Changes Included:
- added hoppy configuration

## How to test this PR
```
cd abd-vro/domain-ee/ee-ep-merge-app
pytest
```

Part Three of breaking down #2089

### CODE REVIEW

**Review by msnwatson — COMMENTED** [review]
do we need any tests for this code?

**Comment by dfitchett** [comment]
@msnwatson there are some tests coming in another PR, and the hoppy library is pretty thoroughly tested in #2044 👍

**Review by nanotone — APPROVED** [review]
LGTM; just a few minor comments. Approving, but if you do decide to redo the hoppy client startup wait, I'd be curious to see what you come up with.

File: domain-ee/ee-ep-merge-app/src/python_src/api.py [hunk 1]
```diff
@@ -24,11 +27,17 @@ async def lifespan(api: FastAPI):
 
 
 async def on_start_up():
-    pass
+    global hoppy
+
+    hoppy.start_hoppy_clients(asyncio.get_event_loop())
+
+    # Wait for hoppy clients to initialize
+    await asyncio.sleep(2)
```
[the remark this answers is not in the document]
Comment (dfitchett) [reply domain-ee/ee-ep-merge-app/src/python_src/api.py:35]: Yeah! I've got some ideas to update the hoppy library with a fix that will address this. I'm thinking we can do that as a separate PR for both after they are both in develop.

**Review by msnwatson — APPROVED** [review]

**Pull request merged**